cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -196,8 +196,6 @@ private predicate isInvalidFunction(Function func) {
     expr.getEnclosingFunction() = func and
     not exists(expr.getType())
   )
-  or
-  count(func.getEntryPoint().getLocation()) > 1
```

Comment:
Only this part should be reverted. The test should simply be updated.

Reply:
Ok, I can do that.

Reply:
Thanks.